custom/icds_reports/migrations/0096_supervisor_id_to_dashboard_ccs_record.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-02-20 08:28
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0095_institutional_delivery_ccs_record_monthly'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='aggregatebirthpreparednesforms',
+            name='supervisor_id',
+            field=models.CharField(max_length=40, null=True),
+        ),
+        migrations.AddField(
+            model_name='aggregateccsrecorddeliveryforms',
+            name='supervisor_id',
+            field=models.CharField(max_length=40, null=True),
+        ),
+        migrations.AddField(
+            model_name='aggregateccsrecordpostnatalcareforms',
+            name='supervisor_id',
+            field=models.CharField(max_length=40, null=True),
+        ),
+        migrations.AddField(
+            model_name='aggregateccsrecordthrforms',
+            name='supervisor_id',
+            field=models.CharField(max_length=40, null=True),
+        ),
+        migrations.AddField(
+            model_name='aggregatecomplementaryfeedingforms',
+            name='supervisor_id',
+            field=models.TextField(null=True),
```

Comment:
why a text field here and a char field elsewhere?

Reply:
I think I fixed it as you were reviewing.